pkg/rpc/server.go

```diff
@@ -271,6 +246,87 @@ Methods:
 	req.WriteResponse(w, results)
 }
 
+func (s *Server) getrawtransaction(reqParams Params) (interface{}, error) {
+	var resultsErr error
+	var results interface{}
+
+	param0, err := reqParams.ValueWithType(0, "string")
+	if err != nil {
+		resultsErr = err
+	} else if txHash, err := util.Uint256DecodeString(param0.StringVal); err != nil {
+		err = errors.Wrapf(err, "param at index 0, (%s), could not be decode to Uint256", param0.StringVal)
```

Comment:
not sure, that in C# version we have same answer.. 
can you try to make request with invalid Uint256..
I think, it can be replaced with `errInvalidParams`

Reply:
Yes, you are right the error message is not consistent with C# version:

`{"jsonrpc":"2.0","error":{"code":-32602,"message":"Invalid Params","data":"unable to decode 62c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7 to Uint256: expected string size of 64 got 63"},"id":1}`},
I will follow your suggestion. :+1: